DxDispatch/src/dxdispatch/CommandLineArgs.cpp

```diff
@@ -119,6 +119,11 @@ CommandLineArgs::CommandLineArgs(int argc, char** argv)
             "Binds input and output resources are allocated from custom heaps. Write-combined caching and system memory (L0) are used when this is specified.",
             cxxopts::value<bool>()
         )
+        (
+            "g, set_stable_power_state",
```

Comment:
Why "g" for set_stable_power_state? I'd prefer to reserve letters for common command line options where the letter is similar to the full option name.